client/src/components/ElementEditor/Header.scss

```diff
@@ -4,9 +4,13 @@
   justify-content: space-between;
 
   &__title {
+    font-size: 15px;
```

Comment:
It's a shame this has to be 15. We have `$h4-font-size` (14px) and `$font-size-lg` (1.231rem / 16px)